mne/epochs.py

```diff
@@ -386,12 +388,17 @@ def decimate(self, decim, copy=False):
                           'result in a sampling frequency of %g Hz, which can '
                           'cause aliasing artifacts.'
                           % (lowpass, decim, new_sfreq))  # > 50% nyquist limit
+        if jitter >= decim:
+            warnings.warn('The jitter=%i is not lower than the decim=%i '
+                          'parameter. Some decimated samples will be lost'
+                          % (jitter, decim))
```

Comment:
Should probably throw an error here. People should use `crop` if they want this behavior.